spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/web/embedded/TomcatWebServerFactoryCustomizerTests.java

```diff
@@ -106,6 +114,14 @@ public void customMaxConnections() {
 	@Test
 	public void customMaxHttpPostSize() {
 		bind("server.tomcat.max-http-post-size=10000");
+		bind("server.tomcat.max-http-form-post-size=10000");
```

Comment:
No need to add this extra property binding as we want to test that the old property still works.